soweego/wikidata/api_requests.py

```diff
@@ -78,9 +79,105 @@ def _lookup_label(item_value):
     if not labels:
         LOGGER.info('No label for %s', item_value)
         return None
+
     return _return_monolingual_strings(item_value, labels)
 
 
+def _process_bucket(bucket, request_params, url_pids, ext_id_pids_to_urls, qids_and_tids,
+                    no_labels_count, no_aliases_count, no_descriptions_count,
+                    no_sitelinks_count, no_links_count, no_ext_ids_count, no_claims_count,
+                    needs_occupations) -> List[Dict]:
+    """
+    This function will be consumed by the `get_data_for_linker`
+    function in this module. It allows the buckets coming from
+    wikidata to be processed in parallel.
+    """
+    response_body = _make_request(bucket, request_params)
+
+    if not response_body:
+        return ''
```

Comment:
Why are you returning an empty string here?

Reply:
good catch! This is a mistake, it should actually return an empty list. It has been fixed in f87f8c681fc5e8c41bbe7e90b7896062aa71ccdf